snippets/models.py

```diff
@@ -501,3 +501,34 @@ def __str__(self):
 
 
 register_snippet(ContentWarning)
+
+class PageLayout(TranslatableMixin, models.Model):
+    LAYOUT_CHOICES = (
+        ('default', 'Default'),
+        ('splash_image', 'Splash Image'),
```

Comment:
what are you imagining these LAYOUT_CHOICES mapping to? i think the only thing we really needed in the layout is the confetti image

the difference in hero type between the homepage and assignable would be in the hero block i think, if thats what you're getting at

Reply:
This is remain consistent in the API, if/when there are other page types.
It'll need some type of identifier, and I don't think we want that to just be a text field. (`layout` below).
I'll remove the `splash_image` and just leave an option for Default.

Reply:
could call it "zebra confetti"